### ISSUE TITLE
New calendar page

### ISSUE THREAD

**Issue by garg3133** [garg3133/JagratiWebApp#122]
A calendar page is required which can be used to display all the events and classes scheduled on a particular day.

Basically the idea is to have two drop-down menus at the top of the page to select the month and year for which we want to see the calendar. The calendar displayed should show all the dates of that month in square blocks (like any usual calendar) with green background if class is scheduled on that day and red background if class is not scheduled.

Only this much work needs to be done for this issue. Rest of the work (displaying all the classes and events scheduled on that day) can be taken in some other issue.

**Comment by micksharma** [comment]
I want to work on this issue.

**Comment by garg3133** [comment]
@micksharma Instead of commenting on 5 different issues, can you please stick with one and let me know which one is it? I'll assign that issue to you.

**Comment by micksharma** [comment]
@garg3133 I am working on this issue. Can you assign it to me.

**Comment by micksharma** [comment]
Thanks for assign me . I commenting because by contributing guidelines first come first serve.

**Comment by garg3133** [comment]
@micksharma Any progress on this?

**Comment by preity-p** [comment]
Hi, can I work on this issue as a GSSoC participant?

**Comment by garg3133** [comment]
Sure @preity-p, go ahead! Assigning you the issue.

**Comment by preity-p** [comment]
> Sure @preity-p, go ahead! Assigning you the issue.

Thank you, I'll start working on it.

**Comment by SmritiAgrawal04** [comment]
May I work on this issue please. Thanks.

**Comment by garg3133** [comment]
Yeah sure @SmritiAgrawal04, go ahead! There is already some work done on this in #188, so you can continue after that.

**Comment by SmritiAgrawal04** [comment]
Yeah, I'm working.
